sidebars.js

```diff
@@ -365,7 +365,7 @@ module.exports = {
   backtrace: [
     {
       type: 'category',
-      label: 'Error Reporting',
+      label: 'Crash and Error Reporting',
```

Comment:
Can we name it Error and Crash Reporting instead of Crash and Error Reproting?
Or is there a reason for Crash coming first?

Reply:
Changed instances of "crash and error" to "error and crash".